doc/python2_support.rst

```diff
@@ -0,0 +1,42 @@
+.. _python2_support:
+
+Python 2 Support Plan for Pydicom
+=================================
+
+.. rubric:: Timeline of support for python 2
+
+The python developers have stated that Python 2 will no longer be supported starting Jan 1, 2020.
```

Comment:
Nitpicking again: Python (the language) is [always written in upper case](https://github.com/python/devguide/blob/master/documenting.rst).

Reply:
Huh, didn't know that was the style - will definitely respect that guide.